CMakeLists.txt

```diff
@@ -4,6 +4,20 @@ if (CMAKE_BINARY_DIR STREQUAL CMAKE_SOURCE_DIR)
     message(FATAL_ERROR "Building in-source is not supported! Create a build dir and remove ${CMAKE_SOURCE_DIR}/CMakeCache.txt")
 endif()
 
+# Note: Searching for CCache must be before project() so project() can use CCache too
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
+
+find_package(CCache)
+
+if (CCache_FOUND)
+    option(OPENRCT2_ENABLE_CCACHE "Enable CCache build cache" ON)
```

Comment:
The description is misleading. It makes user think that such explicit support from build system is required to use ccache. Please make the explanation more correct.